django/views/generic/base.py

```diff
@@ -151,13 +153,21 @@ def get_template_names(self):
 class TemplateView(TemplateResponseMixin, ContextMixin, View):
     """
     A view that renders a template.  This view will also pass into the context
-    any keyword arguments passed by the url conf.
+    any keyword arguments passed by the URLConf.
```

Comment:
It was asked by @timgraham . I guess it's for consistency with the documentation.